addons/base_vat/models/res_partner.py

```diff
@@ -257,7 +258,7 @@ def _build_vat_error_message(self, country_code, wrong_vat, record_label):
             company = self.env.company
 
         vat_label = _("VAT")
-        if country_code and company.country_id and country_code == company.country_id.code.lower():
+        if country_code and company.country_id and country_code == company.country_id.code.lower() and company.country_id.vat_label:
```

Comment:
Is there a reason for this? I mean, you would change the label for all localization, even if it's not a VAT for them, and Tax ID is more suitable for them.
If it's just for Brazil, shouldn't we add it in res_country_data instead?

Reply:
This is a bit of an unrelated change. Basically, If the new user has the same country as the current company, we replace 'VAT' by the country's actual `vat_label`. However, if that country doesn't have a `vat_label` defined, `False` is displayed instead.

Reply:
Indeed sorry, I misread the message